doc/api/repl.markdown

```diff
@@ -64,6 +64,13 @@ the following values:
    returns the formatting (including coloring) to display. Defaults to
    `util.inspect`.
 
+ - `replMode` - controls whether the repl runs all commands in strict mode,
+   default mode, or a hybrid mode ("magic" mode.) Acceptable values are:
+  * `repl.REPL_MODE_DEFAULT` - Run commands in default mode.
+  * `repl.REPL_MODE_STRICT` - Run commands in strict mode.
+  * `repl.REPL_MODE_MAGIC` - Attempt to run commands in default mode. If they
+  fail to parse, re-try in strict mode.
```

Comment:
Maybe put a note that strict mode is the same as 'use strict';? Also should probably not capitalize 'Run' and 'Attempt' here.